### ISSUE TITLE
Export analysis settings

### ISSUE THREAD

**Issue by dsilvestro** [AntonelliLab/raxmlGUI#94]
export file with 
- substitution models, partitions, input file names
- seeds, bootstrap replicates, ML searches
-RAXML and raxmlGUI version

**Comment by jtklein** [comment]
First iteration of this is implemented. Please check if this can be closed.

**Comment by jtklein** [comment]
@dsilvestro Currently, I have something like this as settings file export. Is this okay?

> Your analaysis was run as follows:
Analysis: ML tree inference
Binary: raxml-ng version 1.0.0
Results saved to: /Users/johannesklein/Documents/raxml/TestRuns
RAxML was called with these arguments:
1.) --msa /Users/johannesklein/Documents/raxml/static/example-files/fasta/aminoacid.txt --model PMB --prefix /Users/johannesklein/Documents/raxml/TestRuns/aminoacid --seed 123456 --tree rand{1}

**Comment by dsilvestro** [comment]
yeah I think we should add raxmlGUI's version and the citations of the relevant programs, depending on whether you use RAxML or RAxML-ng and on whether you use model testing. Is this a menu option?
